src/forward/init.jl

```diff
@@ -30,6 +29,16 @@ function ocn_init(Config_filepath; backend=KA.CPU())
     return Setup, Diag, Tend, Prog
 end 
 
+function ocn_init_shadows(Prog, Diag, Tend; backend=KA.CPU())
+
+    d_Prog = PrognosticVars(zeros(Float64, size(Prog.ssh)),
+                            zeros(Float64, size(Prog.normalVelocity)),
+                            zeros(Float64, size(Prog.layerThickness)),
```

Comment:
Just want to make sure the shadow arrays don't need to be/can't be on GPU? If that's the case, then maybe this function doesn't need the `backend` kwarg.

Reply:
The shadow arrays will be on the same device as the primal arrays. That's the advantage of using Enzyme. It's the first AD tool (I know of) that differentiates generic GPU code, where the sensitivities will only involve GPU memory.

Reply:
Oops, this function isn't actually used in the Enzyme tests. It's a holdout from before, we can remove it.